spirekey/src/app/providers.tsx

```diff
@@ -55,7 +55,7 @@ export default function Providers({
               value={{
                 dark: darkThemeClass,
               }}
-              defaultTheme="dark"
+              defaultTheme="light"
```

Comment:
is the system default set by default? Or do we have to tell this provider to use that?

Reply:
`enableSystem` was disabled a few lines above this. I've enabled it now (that's the default) so it sets the them to the preference of the user based on the OS. 

I do think we need a way to switch the theme manually though, @isaozler?